File: mycollab-web/src/main/scala/com/mycollab/module/crm/event/DocumentEvent.scala
package com.mycollab.module.crm.event

import com.mycollab.events.ApplicationEvent

/**
  * @author MyCollab Ltd.
  * @since 5.0.3
  */
object DocumentEvent {

  class GotoDashboard(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}

File: mycollab-web/src/main/java/com/mycollab/module/crm/event/DocumentEvent.kt
package com.mycollab.module.crm.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class DocumentEvent {
    class GotoDashboard(source: Any, data: Any?) : ApplicationEvent(source, data)
}